CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v2.3.0
+
+Example Storefront v2.3.0 is a patch update to keep this project in sync with [Reaction v2.3.0](https://github.com/reactioncommerce/reaction) and [reaction-hydra v2.3.0](https://github.com/reactioncommerce/reaction-hydra)
```

Comment:
This isn't a patch, it's a `minor` version

Reply:
updated